tests/test_search.py

```diff
@@ -24,7 +26,89 @@ def test_search_topic():
         assert repo["watchers_count"] >= 0
         assert repo["forks_count"] >= 0
         assert repo["language"].lower() == "python"
-        assert (repo["description"] == None) or repo["description"]
+        assert (repo["description"] is None) or repo["description"]
+        assert repo["full_name"].count("/") >= 1
+        assert repo["html_url"] == "https://github.com/" + repo["full_name"]
+
+
+def test_search_topics():
+    """
+    Test the search functionality for starcli.search.
+    """
+    repos = search(language="python", topics=["open", "source"])
+    for repo in repos:
+        assert repo["stargazers_count"] >= 0
+        assert repo["watchers_count"] >= 0
+        assert repo["forks_count"] >= 0
+        assert (repo["description"] is None) or repo["description"]
+        assert repo["full_name"].count("/") >= 1
+        assert repo["html_url"] == "https://github.com/" + repo["full_name"]
+
+
+def test_search_created_date():
+    """
+    Test the search functionality for starcli.search.
+    """
+    date_format = "%Y-%m-%d"
+    day_range = 0 - randint(100, 400)
+    created_date_value = (datetime.utcnow() + timedelta(days=day_range)).strftime(
+        date_format
+    )
+    repos = search(language="python", date_created=created_date_value)
+    for repo in repos:
+        assert repo["stargazers_count"] >= 0
+        assert repo["watchers_count"] >= 0
+        assert repo["forks_count"] >= 0
+        assert (repo["description"] is None) or repo["description"]
+        assert repo["full_name"].count("/") >= 1
+        assert repo["html_url"] == "https://github.com/" + repo["full_name"]
+
+        # Need to account for min and max created dates
+        repo_created_date = datetime.strptime(
```

Comment:
I think you could try making this what it was previously, but use pushed_at instead